### ISSUE TITLE
gen-device-svd.py does not process <dim> fields

### ISSUE THREAD

**Issue by aykevl** [aykevl/tinygo#24]
This TODO should be implemented:

```python
                elif el.nodeName == 'cluster':
                    if el.getElementsByTagName('dim'):
                        continue # TODO
```

**Comment by deadprogram** [comment]
We're going to need this to proceed with implementation of the ADC and PWM interfaces on nrf. I2C might be possible, I'm looking into it.

**Comment by aykevl** [comment]
This should be fixed by 5c79a93c85cbe1e46ba2781f7ae0bec904994cf5.
